test-fixtures/gradle-multi-project/README.md

```diff
@@ -0,0 +1,13 @@
+# Example Gradle Multi Project
+
+An example Gradle project used in the tests.
+
+## Formatting massive files
```

Comment:
You can remove all the text from here and below as it's not relevant for this test-fixture.